src/components/Tooltip/STooltip.vue

```diff
@@ -155,6 +155,19 @@ export default class STooltip extends Mixins(BorderRadiusMixin, DesignSystemInje
 
   mounted (): void {
     this.updateCloseDelay(this.closeDelay)
+    window.addEventListener('focus', this.handleWindowFocus)
+  }
+
+  beforeDestroy (): void {
+    window.removeEventListener('focus', this.handleWindowFocus)
+  }
+
+  handleWindowFocus () {
+    setTimeout(() => {
+      if (this.tooltip && this.tooltip.hide) {
+        this.tooltip.hide()
+      }
```

Comment:
I would like to rewrite it as `this.tooltip?.hide?.()`

Reply:
There is no possibility to use new js features in this version of our ui library

Reply:
Ah, I see